examples/model_evaluation/plot_train_test_split.py

```diff
@@ -142,6 +142,19 @@
 print("When expliciting, with the small typo, are the `X_train`'s still the same?")
 print(np.allclose(X_train_explicit, X_train_explicit_inverted))
 
+# %%
+# Returning a dictionnray instead of positional arguments
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+# %%
+
+from sklearn.linear_model import LogisticRegression
+from skore import EstimatorReport
+
+split_data = skore.train_test_split(X=X, y=y, random_state=42, as_dict=True)
+estimator = LogisticRegression(max_iter=10_000, random_state=42)
```

Comment:
```suggestion
estimator = LogisticRegression(random_state=42)
```
It's ok if it doesn't converge, the shorter this example takes to run, the better!